NanoAnalysis/test/prod/samplesNano_2022EE_MC.csv

```diff
@@ -9,80 +9,80 @@ identifier,process,crossSection=-1,BR=1,execute=True,dataset,prefix,pattern,spli
+TTZZ,,0.001579,,,/TTZZ_TuneCP5_13p6TeV_madgraph-madspin-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v3/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;LEPTON_SETUP=2022;DATA_TAG=postEE,RecoProbabilities.py,xsec from XSDB
 
 
 # ###,,,,,,,,,,,bkg for Z+X studies
-WZto3LNu,,5.27,,,/WZto3LNu_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;PROCESS_ZL=True;LEPTON_SETUP=2022,RecoProbabilities.py,#xsex from twiki: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
-DYJetsToLL,,6275.1,,,/DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EENanoAODv12-forPOG_130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;PROCESS_ZL=True;LEPTON_SETUP=2022,RecoProbabilities.py, #xsec from twiki: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
-DYJetsToLL_forPOG,,6275.1,,,/DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EENanoAODv12-forPOG_130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;PROCESS_ZL=True;LEPTON_SETUP=2022,RecoProbabilities.py, #xsec from twiki: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
-TTto2L2Nu,,98.0,,,/TTto2L2Nu_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;PROCESS_ZL=True;LEPTON_SETUP=2022,RecoProbabilities.py, #xsex from https://twiki.cern.ch/twiki/bin/view/LHCPhysics/TtbarNNLO#Updated_reference_cross_sections times BR
+WZto3LNu,,5.27,,,/WZto3LNu_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;PROCESS_ZL=True;LEPTON_SETUP=2022;DATA_TAG=postEE,RecoProbabilities.py,#xsex from twiki: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
+DYJetsToLL,,6275.1,,,/DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EENanoAODv12-forPOG_130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;PROCESS_ZL=True;LEPTON_SETUP=2022;DATA_TAG=postEE,RecoProbabilities.py, #xsec from twiki: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
+DYJetsToLL_forPOG,,6275.1,,,/DYJetsToLL_M-50_TuneCP5_13p6TeV-madgraphMLM-pythia8/Run3Summer22EENanoAODv12-forPOG_130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=1.;GENBR=1.;PROCESS_CR=True;PROCESS_ZL=True;LEPTON_SETUP=2022;DATA_TAG=postEE,RecoProbabilities.py, #xsec from twiki: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
```

Comment:
ciao @AlessandroTarabini , sometimes `DATA_TAG` is `post_EE` and sometimes `postEE`. I think this inconsistency isn't expected and should be fixed?

Reply:
Ciao @bonanomi, you're right. It is fixed now ;)